test/unitTest/java/org/thoughtcrime/securesms/messagehash/MessageHashTest.java

```diff
@@ -0,0 +1,61 @@
+package org.thoughtcrime.securesms.messagehash;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.thoughtcrime.securesms.BaseUnitTest;
+import org.thoughtcrime.securesms.util.MessageHash;
+
+import java.util.HashMap;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertNull;
+
+public class MessageHashTest extends BaseUnitTest {
+    private int    iterationMax = 10000;
+    private String testAddress  = "+15140009999";
+    private String timestamp    = "1231242143";
+    private String expectedHash = "";
+
+
+    @Before
+    public void setup() {
+        this.expectedHash = MessageHash.generateFrom(this.testAddress, this.timestamp);
+    }
+
+    @Test
+    public void testMessageHashGeneratorConsistency() {
+        String actualHash = MessageHash.generateFrom(this.testAddress, this.timestamp);
+
+        assertEquals(actualHash, this.expectedHash);
+    }
+
+    @Test
+    public void testMessageHashGeneratorConsistencyAfterMultipleTries() {
+        int iteration = 0;
+        String[] generatedHashes = new String[this.iterationMax];
+        for( ; iteration != this.iterationMax; iteration++) {
```

Comment:
@veken1199 Why this syntax? Why not just initialize iteration = 0 in the for loop? 
Also, missing space between for and (

Reply:
@erinb90  I refactored and forgot to change it, thx